### PR TITLE
[MISC] address issues raised in context of JOSS review

### PR DESCRIPTION
**Pull request by sappelhoff**
related to #305 

this is work in progress

### CODE REVIEW

**Comment by jasmainak** [comment]
@sappelhoff we need to fix the CIs before we can merge PRs ...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,14 +21,23 @@ pypi. Update your installation as follows.
...
+    $ pip install scikit-learn nibabel nilean pybv
+
 Finally, it is necessary to install the
 [BIDS validator](https://github.com/bids-standard/bids-validator). The outputs
 of MNE-BIDS are run through the BIDS validator to check if the conversion
 worked.
+You will need the `command line version` of the validator, to be installed via
+[Node.js](https://nodejs.org/en/).
+Please follow the installation instructions as detailed in the README of the
+bids-validator repository.
```
Comment (jasmainak) [inline CONTRIBUTING.md]: I think we need people to sometimes use the development version of the validator. It's better to explicitly link [here](https://github.com/bids-standard/bids-validator/blob/master/CONTRIBUTING.md#using-the-development-version-of-bids-validator) for instructions how to install it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,14 +21,23 @@ pypi. Update your installation as follows.
...
 
-Then, install the following python packages:
+Then, install the following python packages for testing purposes:
 
     $ pip install flake8 pytest pytest-cov
 
+Afterwards, also install all optional dependencies of `mne-bids` and their
+dependencies:
+
+    $ pip install scikit-learn nibabel nilean pybv
```
Comment (jasmainak) [inline CONTRIBUTING.md]:
```suggestion
    $ pip install scikit-learn nibabel nilearn pybv
```

**Comment by jasmainak** [comment]
okay see my PR: https://github.com/mne-tools/mne-bids/pull/307. Let's merge this one after that.

**Comment by sappelhoff** [comment]
Thanks for the review and your fix in #307 @jasmainak! I have rebased and integrated your feedback.

This is good to go from my side.

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Pull request merged**